plugins/xdebug/xdebug.cc

```diff
@@ -409,21 +413,73 @@ XInjectResponseHeaders(TSCont /* contp */, TSEvent event, void *edata)
   return TS_EVENT_NONE;
 }
 
+static bool
+isFwdFieldValue(const char *value, int valueLen, intmax_t &fwdCnt)
+{
+  ts::TextView paramName("fwd");
```

Comment:
`constexpr`? Probably at least `static const`.

Reply:
constexpr would be the same as const in this case.  I'd bet $50 that 'paramName' is eliminated by optimization.  So making it static may actually block optimization.

Reply:
Looks like using static is OK.  I thought the compiler hesitated to optimize out statically allocated data even with local linkage, but it seems no:

https://godbolt.org/g/a16SyJ